src/CarbonAware.CLI/test/integrationTests/Commands/EmissionsCommandTests.cs

```diff
@@ -0,0 +1,86 @@
+using CarbonAware.DataSources.Configuration;
+using NUnit.Framework;
+using System.Text.Json.Nodes;
+
+namespace CarbonAware.CLI.IntegrationTests;
+
+/// <summary>
+/// Tests that the CLI handles and packages various responses from aggregators 
+/// and data sources properly, including empty responses and exceptions.
+/// </summary>
+[TestFixture(DataSourceType.JSON)]
+[TestFixture(DataSourceType.WattTime)]
+public class EmissionsCommandTests : IntegrationTestingBase
+{
+    public EmissionsCommandTests(DataSourceType dataSource) : base(dataSource) { }
+
+    [Test]
+    public async Task Emissions_Help_ReturnsHelpText()
+    {
+        // Arrange
+        var expectedAliases = new[]
+        { 
+            "-l", "--location",
+            "-s", "--start-time",
+            "-e", "--end-time",
+        };
+        
+        // Act
+        var exitCode = await InvokeCliAsync("emissions -h");
+        var output = _console.Out.ToString()!;
+
+        // Assert
+        Assert.AreEqual(0, exitCode);
+        foreach (var expectedAlias in expectedAliases)
+        {
+            StringAssert.Contains(expectedAlias, output);
+        }
+    }
+
+    [Test]
+    public async Task Emissions_OnlyRequiredOptions_ReturnsExpectedData()
+    {
+        // Arrange
+        var end = DateTimeOffset.UtcNow;
+        var start = end.AddMinutes(-30);
+        var location = "eastus";
+        _dataSourceMocker.SetupDataMock(start, end, location);
+
+        // Act
+        var exitCode = await InvokeCliAsync($"emissions -l {location}");
+
+        // Assert
+        var jsonResults = JsonNode.Parse(_console.Out.ToString()!)!.AsArray()!;
+        var firstResult = jsonResults.First()!;
+        
+        Assert.AreEqual(0, exitCode);
```

Comment:
nit. Maybe assert right away line 50. So in case something is wrong, lines 53 or 54 won't failed.